nexus/tests/CMakeLists.txt

```diff
@@ -6,10 +6,15 @@ check_python_reqs(numpy "nexus base" ADD_TEST)
 
 if(ADD_TEST)
   message("Adding Nexus tests")
-  set(TESTLIST "")
-  execute_process(COMMAND ${PROJECT_SOURCE_DIR}/nexus/bin/nxs-test --ctestlist OUTPUT_VARIABLE TESTLIST)
-  #MESSAGE("  Nexus tests: ${TESTLIST}")
-  foreach(TESTNAME ${TESTLIST})
+  file(TIMESTAMP ${PROJECT_SOURCE_DIR}/nexus/bin/nxs-test NEXUS_TESTLIST_TIMESTAMP)
```

Comment:
qmcpack_SOURCE_DIR instead of PROJECT_SOURCE_DIR

Reply:
I made this change.